pkg/apis/pipeline/v1beta1/task_validation.go

```diff
@@ -79,7 +79,7 @@ func validateResults(ctx context.Context, results []TaskResult) (errs *apis.Fiel
 
 // a mount path which conflicts with any other declared workspaces, with the explicitly
 // declared volume mounts, or with the stepTemplate. The names must also be unique.
-func validateDeclaredWorkspaces(workspaces []WorkspaceDeclaration, steps []Step, stepTemplate *StepTemplate) (errs *apis.FieldError) {
+func ValidateDeclaredWorkspaces(workspaces []WorkspaceDeclaration, steps []Step, stepTemplate *StepTemplate) (errs *apis.FieldError) {
```

Comment:
Exported func's comments should start with their name, see [this](https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551#comment-sentences )  and other functions for reference. Same with other functions.

If not you will fail the build check